versioneer.py

```diff
@@ -339,7 +339,6 @@
 
 """
 
-from __future__ import print_function
```

Comment:
Please don't change anything in `versioneer.py`. It's a vendored file from (https://github.com/python-versioneer/python-versioneer) with slight modifications. Additional changes make it harder to pull in changes from the upstream version.

Reply:
Thank you @timhoffm . I have removed `versioneer.py` from my PR entirely.